pkg/plugins/runtime/gateway/testdata/https/23-gateway-route.yaml

```diff
@@ -39,7 +39,34 @@ Clusters:
         edsConfig:
           ads: {}
           resourceApiVersion: V3
-      name: echo-service-5a416c39037aa8f6
+      name: echo-service-86192ff3ecc578b7
```

Comment:
I guess this is probably a case where this leads to creating more clusters than strictly necessary?